pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -445,6 +446,7 @@ func realScheduleMonitor(c *Cluster, mon *monConfig) (SchedulingResult, error) {
 		_, err = c.context.Clientset.CoreV1().PersistentVolumeClaims(c.Namespace).Create(pvc)
 		if err == nil {
 			pvcName = pvc.Name
+			d.Labels["canary_pvc_name"] = pvcName
```

Comment:
Should this label also be applied to both the deployment and the deployment's pod template.

Reply:
Same as said for label "mon_type"